models/_sector/nft/trades/chains/bnb/platforms/nftb_bnb_base_trades.sql

```diff
@@ -0,0 +1,157 @@
+              END AS royalty_fee_percentage
+
+FROM {{ source('nftb_bnb', 'NFT_evt_Transfer')}} tr
+INNER JOIN {{ source('nftb_bnb', 'NFT_call_royaltyInfo')}} r
+ON r.call_tx_hash=tr.evt_tx_hash
+INNER JOIN {{ source('bnb','transactions')}} as tr1
+ON tr1.hash=tr.evt_tx_hash
+{% if not is_incremental() %}
+AND tr1.block_time >= TIMESTAMP '{{project_start_date}}'
+{% endif %}
+{% if is_incremental() %}
+AND tr1.block_time >= date_trunc('day', now() - interval '7' day)
+{% endif %}
+LEFT JOIN (SELECT
+                   bt."from" as address
+                  ,tx_hash
+                  ,cast(SUM(value) as uint256) AS value
+            FROM {{ source ('bnb','traces')}} bt
+            WHERE 1=1
+            AND (bt."from"=0xebd4232e4c1999bc9562802eae01b431d5053e65 --seller contract address when withdraw function is called
+            OR
+            bt."from"=0xdc2f08364ebc6cebe0b487fc47823b1e83ce8550 -- seller contract address when claim function is called
+            )
+            AND bt.input=0x
+            AND bt.call_type='call'
+            {% if not is_incremental() %}
+            AND bt.block_time >= TIMESTAMP '{{project_start_date}}'
+            {% endif %}
+            {% if is_incremental() %}
+            AND bt.block_time >= date_trunc('day', now() - interval '7' day)
+            {% endif %}
+            GROUP BY 1,2
+            ) tra1
+ON tra1.tx_hash=tr.evt_tx_hash
+LEFT JOIN {{ source('bnb','logs')}} l1
+ON l1.tx_hash=r.call_tx_hash
+AND l1.contract_address=0xdc2f08364ebc6cebe0b487fc47823b1e83ce8550
+AND l1.topic0=0x7a9edcf72fda2a090305f15fe2f1d8d881c849c4a142e8847734fe93542c64ef --func sig for claim function call
+{% if not is_incremental() %}
+AND l1.block_time >= TIMESTAMP '{{project_start_date}}'
+{% endif %}
+{% if is_incremental() %}
+AND l1.block_time >= date_trunc('day', now() - interval '7' day)
+{% endif %}
+
+LEFT JOIN {{ source('bnb','logs')}} l2
+ON l2.tx_hash=r.call_tx_hash
+AND l2.contract_address=0xe22c90e7816db4344f33c651c7b0a01fcd51a327
+AND l2.topic0=0x7df6bafa53a0e01e6995efd8c0c627e532d2fb130178b2261d619f256db0d65a --func sig for buy function call
+{% if not is_incremental() %}
+AND l2.block_time >= TIMESTAMP '{{project_start_date}}'
+{% endif %}
+{% if is_incremental() %}
+AND l2.block_time >= date_trunc('day', now() - interval '7' day)
+{% endif %}
+
+LEFT JOIN {{ source('bnb','logs')}} l3
+ON l3.tx_hash=r.call_tx_hash
+AND l3.contract_address=0xebd4232e4c1999bc9562802eae01b431d5053e65
+AND l3.topic0=0x3966e47923a4243aaa12fcf3bb231f645e3c8c5e70985cd00c689ec364cf4da0 --func sig for withdraw function call (Auction Settled)
+{% if not is_incremental() %}
+AND l3.block_time >= TIMESTAMP '{{project_start_date}}'
+{% endif %}
+{% if is_incremental() %}
+AND l3.block_time >= date_trunc('day', now() - interval '7' day)
+{% endif %}
+
+WHERE tr.evt_block_time >= TIMESTAMP '{{project_start_date}}'
+{% if is_incremental() %}
+AND tr.evt_block_time >= date_trunc('day', now() - interval '7' day)
+{% endif %}
```

Comment:
we're considering this model an exception, since it requires base table fields to obtain `amount_raw` values?

Reply:
Yes, it's probably due a proper rewrite to get the base table joins out of the model. But I'd consider that out of scope of this PR.